src/docker/job_control.py

```diff
@@ -176,27 +192,24 @@ def run_job(job: str, s3client: client) -> int:
         [],
     )
 
-    job_output = f"> {job_type}.stdout.txt 2> {job_type}.stderr.txt"
+    job_stdout_filename = f"{job_type}.stdout.txt"
+    job_stderr_filename = f"{job_type}.stderr.txt"
     if JOBTYPE.APBS.name.lower() in job_type:
         command = (
-            "LD_LIBRARY_PATH=/app/APBS-3.0.0.Linux/lib "
             "/app/APBS-3.0.0.Linux/bin/apbs "
```

Comment:
I don't think this will work without setting LD_LIBRARY_PATH ("LD_LIBRARY_PATH=/app/APBS-3.0.0.Linux/lib ").

Reply:
https://github.com/Electrostatics/apbs-aws/blob/Eo300/issue_48/src/docker/Dockerfile#L17
Even when set within the Dockerfile?

Reply:
If that is the case, then the PATH should include /app/APBS-3.0.0.Linux/bin so you don't have to hardcode that path either.

Reply:
Gotcha. Just made the changes